about/index.md

```diff
@@ -4,7 +4,13 @@ title: "What is MEI?"
 ---
 # An introduction to MEI
 
-The Music Encoding Initiative (MEI) is a community-driven effort to define guidelines for encoding musical documents in a machine-readable structure. MEI closely mirrors work done by text scholars in the [Text Encoding Initiative](http://www.tei-c.org/) (TEI) and while the two encoding initiatives are not formally related, they share many common characteristics and development practices. MEI, like TEI, is an umbrella term to simultaneously describe an organization, a research community, and a markup language. It brings together specialists from various music research communities, including technologists, librarians, historians, and theorists in a common effort to discuss and define best practices for representing a broad range of musical documents and structures. The results of these discussions are then formalized into the MEI schema, a core set of rules for recording physical and intellectual characteristics of music notation documents. This schema is developed and maintained by the [MEI Technical Team](/community/technical-team.html).
+The Music Encoding Initiative (MEI) is a 21st century community-driven [open-source](https://github.com/music-encoding/music-encoding) effort to define guidelines for encoding musical documents in a machine-readable structure. 
```

Comment:
"21st century" - sort of goes without saying

Reply:
Otherwise, it looks fine!

Reply:
thanks for this and sorry for late reaction